src/components/TextArea/TextArea.tsx

```diff
@@ -78,13 +78,13 @@ const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
             {label}
           </Label>
         )}
-
-        {errorMessage && (
+        <div role="status">
           <StyledErrorMessage
             id="textareaErrorMessage"
-            message={errorMessage}
+            data-testid="textareaError"
+            message={errorMessage || ''}
           />
```

Comment:
This component should only be rendered when there is an actual error message that needs to be shown.